Airbrakes/airbrakes.py

```diff
@@ -1,10 +1,15 @@
 """Module which provides a high level interface to the air brakes system on the rocket."""
 
-from Airbrakes.imu import IMU, IMUDataPacket
+from typing import TYPE_CHECKING
+
+from Airbrakes.imu import IMU, IMUDataPacket, RollingAverages
```

Comment:
if everything is going to be in an `airbrakes` package you might as well use relative imports `from .imu import IMU, bla`

that's typically the better way IIRC

Reply:
also it doesn't really matter but package names should be lowercase so airbrakes not `Airbrakes`

Reply:
absolute imports are recommended per [PEP-8](https://peps.python.org/pep-0008/#imports), its way more clear

Reply:
agreed to make it lowercase tho